README.md

```diff
@@ -1,14 +1,91 @@
+### Why RheumaConnect?
+Rheumatic patients often face several challenges,
+
+• Limited access to reliable medical information about their conditions.
+• Difficulty in understanding complex medical terminology.
+• Long waiting times for specialist consultations.
+• Confusion about medication and treatment options.
+• Language barriers in accessing medical information.
+
+RheumaConnect is transforming rheumatic care by offering an integrated approach to patient support. The platform provides free access to expert verified medical resources, with easy to understand explanations of conditions and treatments. Patients can quickly book appointments, reducing waiting times, and access clear medication guidelines along with detailed treatment information. With multi-language support, all resources are available in patients' preferred languages, ensuring better understanding and promoting improved healthcare outcomes.
+
+---
+
+## 🌟 Key Features
+
+• Tri-lingual support (English, Sinhala, Tamil).
+• Expert verified medical information.
+• Free medical advice for managing rheumatic conditions.
+• Appointment booking system with rheumatology specialists.
+• Treatment options and medication guides.
+• User-friendly interface designed for accessibility.
+• Comprehensive resource library for various rheumatic conditions.
+• Mobile responsive design for access anywhere, anytime.
+
+---
+
+## 🚀 Current State
+RheumaConnect has successfully completed the design phase and the project is now in the development phase. Currently, the team is focused on building the platform's core features.
+
+### Roadmap
+Phase 1: Website development and initial content population
+Phase 2: Integration of multilingual support
+Phase 3: Continuous content updates and feedback integration
+
+---
+
+## 👥 Project Team
+
+### Project Initiators
+The project was initiated by the Software Engineering Student Community of SLIIT, in collaboration with medical professionals to bring this vision to life.
+
+### Core Team
+Software Engineering Student Community (SESC) - SLIIT
+Medical Specialists (Rheumatology Department)
+Volunteer Developers and Designers
+
+---
+
+## 🤝🏽 How You Can Contribute
+If you are passionate about making a difference and contributing to the project, we would love for you to get involved! You can contribute in several ways,
+
+### Technical Contributions
+
+• Frontend Development: Assist in designing the user interface for an intuitive and user-friendly experience.
+• Backend Development: Help in developing the server-side architecture and integration of multilingual capabilities.
+• UI/UX Design: Contribute to creating a visually appealing and accessible design for the platform.
+• Testing: Help with quality assurance by writing test cases, identifying and fixing bugs.
+
+### Non-Technical Contributions
+
+• Content Creation: Contribute by writing or curating informative articles for the platform, ensuring they are clear, concise, and    medically accurate.
```

Comment:
Update this with 'Content writing' for the website (this would be the static content in the website, not the dynamic ones) The articles and stuff will be written by the doctors only.